packages/payload/src/utilities/createArrayFromCommaDelineated.ts

```diff
@@ -2,8 +2,6 @@ export function createArrayFromCommaDelineated(input: string): string[] {
   if (Array.isArray(input)) {
     return input
   }
-  if (input.indexOf(',') > -1) {
-    return input.split(',')
-  }
-  return [input]
+
+  return input.split(',')
```

Comment:
If input does not contain `,` then split already returns `[input]` => no need for manual checking